paddle/pybind/pybind.cc

```diff
@@ -116,6 +119,47 @@ PYBIND11_PLUGIN(core) {
         return self.data<float>()[offset];
       });
 
+  py::class_<LoDTensor>(m, "LoDTensor", R"DOC(LoD(Leval of Ddetails) Tensor.
+
+The tensor and LoD info should be created before creating the LoDTensor, then
+call the set_tensor and set_lod functions to set them.
+
+)DOC")
+      .def("set_tensor",
+           [](LoDTensor &self, Tensor *tensor) { self.set_tensor(tensor); })
+      .def("set_lod",
+           [](LoDTensor &self, std::vector<std::vector<size_t>> &lod) {
+#ifdef PADDLE_ONLY_CPU
+             self.set_lod(lod);
+#else
+             paddle::framework::LoD new_lod;
+             new_lod.reserve(lod.size());
```

Comment:
expose other interfaces like, `SliceLevels`, `SliceInLevel`, `NumLevels`, `NumElements`, these interfaces may be used by user too.

What's more, `LODSliceInLevel`, `LoDSliceLevels` may be operators also, which may be used in RNN or some other Ops.  These can be new issues.

Reply:
> SliceLevels, SliceInLevel, NumLevels, NumElements

Now, all these interfaces are used in C++ code.  Just like Tensor, many interfaces are not exposed.   I'd like to add these interfaces in the future if they are really needed in Python.